SITnett/templates/produksjoner/produksjoner.html

```diff
@@ -1,10 +1,33 @@
 {% extends "meny.html" %}
+{% load static %}
 
 {% block innhold %}
 
+<div class="card-wrapper">
 	{% for produksjon in pliste %}
-	<a href="{% url 'produksjon_info' produksjon.id %}">{{produksjon}}</a><br>
+	<a class="production-card" href="{% url 'produksjon_info' produksjon.id %}">
+		<img src="{{produksjon.banner.url}}" class="production-card-image">
+		<div class="production-card-container">
+		<h3 class="production-card-title">{{produksjon.tittel}}</h3> 
+		<div class="text-info text-strong">{{produksjon.premiere}}</div> 
+		</div>
+	</a>
+	<a class="production-card" href="{% url 'produksjon_info' produksjon.id %}">
+		<img src="{{produksjon.banner.url}}" class="production-card-image">
+		<div class="production-card-container">
+		<h3 class="production-card-title">{{produksjon.tittel}}</h3> 
+		<div class="text-info text-strong">{{produksjon.premiere}}</div> 
+		</div>
+	</a>
+	<a class="production-card" href="{% url 'produksjon_info' produksjon.id %}">
+		<img src="{{produksjon.banner.url}}" class="production-card-image">
+		<div class="production-card-container">
+		<h3 class="production-card-title">{{produksjon.tittel}}</h3> 
+		<div class="text-info text-strong">{{produksjon.premiere}}</div> 
+		</div>
+	</a>
```

Comment:
Slik dette er satt opp nå kommer det tre identiske kort for hver enkelt produksjon. Antar det bare er for testing?

Reply:
Riktig. Fjerner :)